tests/integration.rs

```diff
@@ -2728,6 +2728,112 @@ mod tests {
             handle_child_output(r, &output);
         }
 
+        #[test]
+        #[cfg(not(feature = "mshv"))]
+        fn test_pci_multiple_segments() {
+            let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+            let guest = Guest::new(Box::new(focal));
+
+            // Prepare another disk file for the virtio-disk device
+            let test_disk_path = String::from(
+                guest
+                    .tmp_dir
+                    .as_path()
+                    .join("test-disk.raw")
+                    .to_str()
+                    .unwrap(),
+            );
+            assert!(exec_host_command_status(
+                format!("truncate {} -s 4M", test_disk_path).as_str()
+            )
+            .success());
+            assert!(
+                exec_host_command_status(format!("mkfs.ext4 {}", test_disk_path).as_str())
+                    .success()
+            );
+
+            let mut cmd = GuestCommand::new(&guest);
+            cmd.args(&["--cpus", "boot=1"])
+                .args(&["--memory", "size=512M"])
+                .args(&["--kernel", direct_kernel_boot_path().to_str().unwrap()])
+                .args(&["--cmdline", DIRECT_KERNEL_BOOT_CMDLINE])
+                .args(&["--platform", "num_pci_segments=2"])
+                .args(&[
+                    "--disk",
+                    format!(
+                        "path={}",
+                        guest.disk_config.disk(DiskType::OperatingSystem).unwrap()
+                    )
+                    .as_str(),
+                    format!(
+                        "path={}",
+                        guest.disk_config.disk(DiskType::CloudInit).unwrap()
+                    )
+                    .as_str(),
+                    format!("path={},pci_segment=1", test_disk_path).as_str(),
```

Comment:
should better be `pci_segment=15`, this will make us test both the first and the last of PCI segment.

Reply:
Done